src/components/table-hoc/TableRowConnected.tsx

```diff
@@ -171,6 +174,12 @@ class TableRowConnected extends React.PureComponent<ITableRowConnectedProps & Re
             this.props.onClick(isMulti);
         }
     }
+
+    private handleDoubleClick = () => {
+        _(this.props.actions)
+            .filter((action: IActionOptions) => action.callOnDoubleClick)
+            .forEach((action: IActionOptions) => action.trigger());

```

Comment:
is there really a case where we want more than one action to be triggered ?

Reply:
clean style by the way, you could do it without underscore too

Reply:
I don't know if there really is a use case for it, but that's the same logic we implemented on the previous table component.